src/ripple/nodestore/impl/Shard.h

```diff
@@ -170,6 +174,8 @@ class Shard final
     Application& app_;
     mutable std::recursive_mutex mutex_;
 
+    DatabaseShard const& db_;
```

Comment:
Needs comment explaining why the lifetime of `db_` outlives this object (Note to self: look at lifetime issue)

Reply:
Done. db_ excluded.